src/Microsoft.AspNet.OData.Shared/Query/Expressions/SelectExpandIncludeProperty.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using Microsoft.AspNet.OData.Common;
+using Microsoft.OData.Edm;
+using Microsoft.OData.UriParser;
+
+namespace Microsoft.AspNet.OData.Query.Expressions
+{
+    internal class SelectExpandIncludeProperty
```

Comment:
>SelectExpandIncludeProperty [](start = 19, length = 27)

If this represents an included property in the SelectExpand, maybe call this SelectExpandIncludedProperty (with a "d")